src/Chapter3.hs

```diff
@@ -971,6 +1071,31 @@ implement the following functions:
 🕯 HINT: to implement this task, derive some standard typeclasses
 -}
 
+{- ORMOLU_ENABLE -}
+data Weekday = Monday | Tuesday | Wednesday | Thursday | Friday | Saturday | Sunday deriving (Show, Eq, Ord, Enum)
+
+isWeekend :: Weekday -> Bool
+isWeekend x
+  | x == Saturday || x == Sunday = True
+  | otherwise = False
+
+nextDay :: Weekday -> Weekday
+nextDay Sunday = Monday
```

Comment:
That is right!
But you could use the instance of `Bounded` here:

```suggestion
    | day == maxBound = minBound
```

This is also a bit more maintainable. If you decide, for example, to have the "American" system of week days in the data type constructors, you won't need to change this function at all! 🙂